### PR TITLE
Change recommended plugins

### PR DESCRIPTION
**Pull request by cristian-ungureanu**
Changed the recommending plugins according to https://github.com/Codeinwp/themeisle-companion/issues/733

Testing scenarios:
1. Test if the newly added plugins can be installed. activated, deactivated from there.
2. Check if the link for the premium plugin is working.

### CODE REVIEW

File: core/app/class-orbit-fox-admin.php [hunk 1]
```diff
@@ -411,14 +411,24 @@ public function get_recommended_plugins() {
...
+			'wp-maintenance-mode',
+			'multiple-pages-generator-by-porthas',
+		];
+
+		$th_plugins = [
+			'wp-landing-kit' => [
+				'banner'      => esc_url( OBFX_URL ) . '/dashboard/assets/wp-landing.jpg',
+				'name'        => 'WP Landing Kit',
+				'description' => __( 'Turn WordPress into a landing page powerhouse with Landing Kit. Map domains to pages or any other published resource.', 'themeisle-companion' ),
+				'author'      => 'ThemeIsle',
```
Comment (selul) [inline core/app/class-orbit-fox-admin.php:426]:
```suggestion
				'author'      => 'Themeisle',
```
It should be `Themeisle` with a small `i`.

**Review by selul — CHANGES_REQUESTED** [review]

**Comment by irinelenache** [comment]
@cristian-ungureanu Tested and here are my findings:

- [ ] I can't install the recommnended plugins using this PR (it works fine with the wp.org version of obfx) https://vertis.d.pr/GgaZty
- [ ] For WP Maintenance mode there is an escaped character here https://vertis.d.pr/4dhxnq

The issues can be checked here:
```
 Admin area URL: https://righteousvacation.tastewp.com/wp-admin 
 Username: irinel 
 Password: YN2XYSHgmZo 
```

**Comment by cristian-ungureanu** [comment]
Thank you @irinelenache, it should work now 🚀

**Comment by irinelenache** [comment]
@cristian-ungureanu Tested again and the reported issues are fixed, thank you 🚀

File: dashboard/src/scss/components/_plugin-card.scss [hunk 2]
```diff
@@ -36,6 +47,10 @@
       display: flex;
       align-items: center;
     }
+
+    .components-external-link{
```
Comment (UVLabs) [inline dashboard/src/scss/components/_plugin-card.scss:51]: @cristian-ungureanu I don't think we need this extra style here, we can just add the `plugin-action` class to the component, it has the style already: https://github.com/Codeinwp/themeisle-companion/blob/f09a67da20dcaa95d5337ad8553d73c7094848cf/dashboard/src/scss/components/_plugin-card.scss#L69

Comment (cristian-ungureanu) [reply dashboard/src/scss/components/_plugin-card.scss:51]: Nice catch! Can you create another issue, please? I already released it and we'll forget to update next time.

**Pull request merged**